src/services/outliningElementsCollector.ts

```diff
@@ -33,7 +33,7 @@ namespace ts.OutliningElementsCollector {
             const currentLineStart = lineStarts[i];
             const lineEnd = i + 1 === lineStarts.length ? sourceFile.getEnd() : lineStarts[i + 1] - 1;
             const lineText = sourceFile.text.substring(currentLineStart, lineEnd);
-            const result = lineText.match(/^\s*\/\/\s*#(end)?region(?:\s+(.*))?$/);
+            const result = lineText.match(/^\s*\/\/\s*#(end)?region(?:\s+(.*))?(?:\r)?$/);

```

Comment:
Is it necessary to match the rest of the line? Could the regex just stop at `region`?

Reply:
It is necessary, I need the label to display when folded.

Reply:
I think @andy-ms meant `/^\s*\/\/\s*#(end)?region(?:\s+(.*))?/` which still captures the label, right?